File: python/problem_one.py
from itertools import count, takewhile
import operator as op

def multiple_of(n):
    return lambda x: x % n == 0

def main():
    s = sum((x for x in range(1, 1000) 
        if multiple_of(3)(x) or multiple_of(5)(x)))         
    print(s)
    


     

if __name__ == '__main__':
    main()

File: javascript/problem-one.js
const { or, sum, range, multipleOf } = require('./utils');

function main() {
	const s = sum(
		Array.from(range(1, 1000))
		     .filter(or(multipleOf(3), multipleOf(5)))
	)
	console.log(s);
}

main();
